extension/EdFi.Ods.Extensions.LMSX/LMS-Harmonizer/1090-lms-assignment-submissions.sql

```diff
@@ -73,29 +69,32 @@ BEGIN
 	FROM edfi.StudentSectionAssociation edfisectionassociation
 	INNER JOIN lmsx.Assignment lmsxassignment
 		ON edfisectionassociation.SectionIdentifier = lmsxassignment.SectionIdentifier
+	INNER JOIN lms.Assignment lmsassignment
+		ON lmsassignment.SourceSystemIdentifier = lmsxassignment.AssignmentIdentifier
 	INNER JOIN edfi.Student edfistudent
 		ON edfistudent.StudentUSI = edfisectionassociation.StudentUSI
 	INNER JOIN lms.LMSUser lmsstudent
 		ON lmsstudent.EdFiStudentId = edfistudent.Id
 	INNER JOIN edfi.Section edfisection
-		ON edfisection.SectionIdentifier = lmsxassignment.SectionIdentifier
+		ON edfisection.SectionIdentifier = edfisectionassociation.SectionIdentifier
```

Comment:
When joining LMS data to EdFi sections, we have made the assertion that Section Identifier needs to be unique, and elsewhere the harmonizer requires that.

It feels weird to have only this one key, when normally a join between `StudentSectionAssociation` and `Section` would _require_ that you use all elements of the natural key (`LocalCourseCode`, `SchoolId`, and `SchoolYear` being the other columns, I think).

I'm comfortable using only the one column instead of all of them. But it might be useful to add a comment for future readers, to know that this was a deliberate decision and not an oversight.

```suggestion
        # The LMS Harmonizer requires that SectionIdentifier be unique, thus it is
        # safe in this scenario to ignore the other natural key elements in this join.
		ON edfisection.SectionIdentifier = edfisectionassociation.SectionIdentifier
```